doc/design/restore_existing.md

```diff
@@ -0,0 +1,17 @@
+Restore From Existing Backup
+======
+
+We want to restore an etcd cluster from existing backup that operator has saved before.
+
+We add a field in BackupPolicy:
+```Go
+type BackupPolicy struct {
+    // InitPV...
+    InitPV string `json:"initPV"`
```

Comment:
in the backupolicy struct, we do not have any PV specific field. I think we should avoid it if possible.

is there a reason that we need a string here? I feel a bool like `recoveryFromBackup`  in the cluster spec outside backup would be good enough?